samples/boards/stm32/power_mgmt/blinky/prj.conf

```diff
@@ -1,5 +1,4 @@
 CONFIG_PM=y
 CONFIG_PM_DEVICE=y
 CONFIG_PM_DEVICE_RUNTIME=y
-CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=n
```

Comment:
To be replaced by `PM_DEVICE_SYSTEM_MANAGED=y` then ?

Reply:
that is enabled by default.